### PR TITLE
Contribution hotfix

### PR DESCRIPTION
**Pull request by benghaziboy**

### CODE REVIEW

**Comment by benghaziboy** [comment]
@mparent61

File: bulbs/contributions/views.py [hunk 1]
```diff
@@ -165,6 +165,7 @@ def list(self, request):
         format = self.request.QUERY_PARAMS.get('format', None)
         if format == 'csv':
             queryset = self.get_queryset()
+            queryset = ESPublishedFilterBackend().filter_queryset(request, queryset, None)
```
Comment (mparent61) [inline bulbs/contributions/views.py]: Could there be any other future surprises from not calling super()?

**Comment by mparent61** [comment]
Changelog!

**Comment by mparent61** [comment]
:+1: With an appropriate and eloquent changelog

File: bulbs/contributions/views.py (hunk 1 above)
Comment (benghaziboy) [inline bulbs/contributions/views.py]: Possibly? I would like to rewrite this implementation so we aren't separating anything about the csv & api responses until the end, let that happen somewhere else.

TODO added.

**Pull request merged**